ibex_demo_system_core.core

```diff
@@ -21,7 +22,12 @@ filesets:
       - rtl/system/spi_top.sv
     file_type: systemVerilogSource
 
+  files_lint_verilator:
+    files:
+      - lint/verilator_waiver.vlt: {file_type: vlt}
```

Comment:
Should this point to the `demo_system_verilator_lint.vlt` file created in this commit?